edb/pgsql/compiler/dml.py

```diff
@@ -144,7 +144,7 @@ def init_dml_stmt(
     ):
         dml_cte = pgast.CommonTableExpr(
             query=pgast.SelectStmt(),
-            name=ctx.env.aliases.get(hint='m')
+            name=ctx.env.aliases.get(hint='melse')
```

Comment:
Is this hunk intentional?

Reply:
Yeah, I was trying to make sure I understood where the `m`s were coming from